NHSN/src/test/java/com/nhsn/pages/ReviewPage.java

```diff
@@ -54,6 +59,36 @@ public void clickOnGenerateButton()
         System.out.println("Clicked on Generate Button");
     }
 
+    public void enterSubmitDateOnReviewPage(String date) throws InterruptedException {
+
+        WebElement submitDateTextbox = driver.findElement(By.xpath("//input[@name='dp']"));
+        char dateArray[] = date.toCharArray();
+        for(int i=0;i< dateArray.length;i++)
+        {
+            submitDateTextbox.sendKeys(Character.toString(dateArray[i]));
+            Thread.sleep(1000);
+        }
+        System.out.println("Entered the date as -"+ date);
+    }
+
+    public void verifySubmittedReportOnReviewPage( String note)  {
```

Comment:
Change `note` param to `text`. I believe this is just looking for any text in a span in the table of the review screen? Not looking specifically for the "note" text to match anything specific...

Reply:
Done